frontend/src/pages/Landing.jsx

```diff
@@ -88,7 +96,7 @@ export default function Landing() {
             <OptionButton
               glyph="import"
               header="Answer Explorer"
-              text="Easily upload JSON files of answersets to view them in Robokop&apos;s graphical interface."
+              text={`Easily upload JSON files of answersets to view them in ${brandConfig.title}'s graphical interface.`}
```

Comment:
Does the apostrophe work okay here now? I saw that before you had to use \&apos;

Reply:
Yeah. I think the apostrophe would have actually worked in the old format as well, the &apos might have been when that text was directly in the html. The new backtick allows normal apostrophes.